app/core/src/screens/hotels/AllHotelsNavigationScreen.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { TouchableOpacity } from 'react-native';
+import {
+  AllHotels,
+  type SearchParametersType,
+} from '@kiwicom/react-native-app-hotels';
+import { Ionicons } from '@expo/vector-icons';
+
+import type { Navigation } from '../../types/Navigation';
+
+type Props = {|
+  navigation: Navigation,
+  hotelsFilter: SearchParametersType,
+  onFilterChange: SearchParametersType => void,
```

Comment:
I have the same objection as on the `AllHotelsMapNavigationScreen` component about the props type control.